acceptance/tests/vault/vault_test.go

```diff
@@ -88,17 +103,23 @@ path "consul/data/secret/gossip" {
 	}
```

Comment:
why don't we need to generate anything for the connect ca like we generated the gossip secret? is the only info vault needs in this case just the address and two paths and it will generate the certs itself? maybe its worth a comment here but also if this is just something that would be documented in user facing docs that's fine too.

Reply:
>is the only info vault needs in this case just the address and two paths and it will generate the certs itself?
Yep, except it's Consul that will generate it for us (as long as the policy allows it to do so).

From https://www.consul.io/docs/connect/ca/vault#root-and-intermediate-pki-paths:

> If either path does not exist, then Consul will attempt to mount and initialize it. This requires additional privileges by the Vault token in use. If the paths already exist, Consul will use them as configured.